tests/schemas/pilot.schema.json

```diff
@@ -10,6 +10,13 @@
     "xws": { "type": "string", "pattern": "^[a-z0-9-]+$" },
     "ability": { "type": "string" },
     "text": { "type": "string" },
+    "standardLoadout": { "type": "boolean"},
+    "upgrades": {
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
Is there a way we can point this towards the list of upgrades, to validate that all upgrades exist in upgrades/*.json?

Reply:
I don't think this can be done in the schema, because you'd first have to load the list of upgrades into memory and then compare these entries against them.

Another example of a test too complex to live purely in schema is detection of duplicate XWS ids: https://github.com/guidokessels/xwing-data2/blob/master/tests/xws.test.js

Possible solutions here:
1. no check (not ideal, but error impact is fairly low)
2. check that entries match the rough XWS regex (see line 10, `"xws": { "type": "string", "pattern": "^[a-z0-9-]+$" },`)
3. Write another test like `xws.text.js` which loads all the valid upgrade xws IDs into an array and then matches each of the standardLoadout upgrade Ids against that array

(3) is clearly ideal but involves writing js, and (2) is probably sufficient if you are pressed for time. Up to you which of (2) and (3) you have capacity to implement.

Reply:
Actually, after thinking further, just do (2). All the errors I saw in this PR were typing the wrong upgrade, not a typo in the upgrade name. So the regex should be plenty